test/datalog_tests/tutorial.ast.expected

```diff
@@ -270,6 +270,7 @@ function std.result_unwrap_or (res: std.Result<'V,'E>, def: 'V): 'V =
         std.Ok{.res=var v} -> v,
         std.Err{.err=_} -> def
     }
+extern function std.result_unwrap_or_default (res: std.Result<'V,'E>): 'V
```

Comment:
I wonder whether we could somehow automatically prepend std to avoid always changing all reference outputs